packages/mopsa/mopsa.1.2/opam

```diff
@@ -0,0 +1,108 @@
+opam-version: "2.0"
+synopsis:
+  "MOPSA: A Modular and Open Platform for Static Analysis using Abstract Interpretation"
+description: """\
+MOPSA is a generic framework for building sound static analyzers based on Abstract Interpretation.
+It features a modular architecture to support different kinds of languages, iterators, and abstract domains.
+For the moment, MOPSA can analyze programs written in a subset of C and Python.
+It reports run-time errors on C programs and uncaught exceptions on Python programs."""
+maintainer: [
+  "Antoine Miné <antoine.mine@lip6.fr>"
+  "Abdelraouf Ouadjaout <ouadjaout@gmail.com>"
+  "Raphaël Monat <raphael.monat@inria.fr>"
+]
+authors: [
+  "Antoine Miné"
+  "Abdelraouf Ouadjaout"
+  "Matthieu Journault"
+  "Aymeric Fromherz"
+  "Raphaël Monat"
+  "Francesco Parolini"
+  "Marco Milanese"
+  "Jérôme Boillot"
+]
+license: "LGPL-3.0-or-later"
+homepage: "https://mopsa.lip6.fr"
+doc: "https://mopsa.gitlab.io/mopsa-analyzer/user-manual/"
+bug-reports: "https://gitlab.com/mopsa/mopsa-analyzer/issues"
+depends: [
+  "ocaml" {>= "4.13.0"}
+  "dune" {>= "3.7"}
+  "ocamlfind"
+  "apron" {>= "0.9.15"}
```

Comment:
```suggestion
  "apron" {>= "v0.9.15"}
```

`apron` uses a `v`-prefixed naming scheme (for the past 6 years).
In #28224 I propose the same tweak for the 1.0 and 1.1 releases.

Reply:
Thanks for catching this, @jmid! I've also noticed similar problems with dependencies on `base`/`core` in the past. Should this be something that's caught by the lint checks? :thinking: /cc @shonfeder

Reply:
Yes, I think so! It is a common enough error to warrant a lint check (#28224 is just one of the first from a bunch of findings I have lined up).

Could it be as simple as "check first character in bound for a matching first character in a release"? :thinking: